src/views/PremiumChoices.vue

```diff
@@ -1,126 +1,103 @@
 <template>
   <div class="container">
-<div class="row">
-  <div class="col-md-12 text-center">
-    <h1>Subscribe</h1>
+    <div class="row">
+      <div class="col-md-12 text-center">
+        <h1>Subscribe</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-12 text-center">
+        <h3 class="subtitle">Become a paid subscriber and get ad-free episodes</h3>
+      </div>
+    </div>
+
+    <div class="row text-center">
+      <div class="col-md-offset-2 col-md-6 text-center">
+        <h4 class="plan-name"> Monthly </h4>
+        <h4 class="price">
+          <span class="dollars">$10</span> /mo
+        </h4>
+        <span class="billed-terms">billed monthly</span>
+        <button @click="signUpForSubscriptionMonthly" class="monthly-button">
+          Get Started
+        </button>
+      </div>
+
+      <div class="col-md-5 text-center yearly-plan">
+        <h4 class="plan-name">Yearly</h4>
+        <h4 class="price">
+          <span class="dollars">$100</span> /yr
+        </h4>
+        <span class="billed-terms">billed yearly</span>
+        <button @click="signUpForSubscriptionYearly" class="yearly-button">
+          Get Started
+        </button>
+      </div>
+    </div>
+
+    <div class="row why-subscribe">
+      <div class="col-md-12 text-center why-subscribe-title">
+        <h3>Why Subscribe?</h3>
+      </div>
+      <div class="col-md-12 why-subscribe-inner">
+        <br />
```

Comment:
I recommend not using <br> and using more CSS

Reply:
I agree with your recommendation. There are just a lot of these side pages with not the greatest HTML formatting. Lots of `<br />` tags. I didn't want to waste a lot of timing refactoring this entire page. I'll clean this page up a bit.

Reply:
I understand you completely. Do it only if you think it's worth it at the moment. We can't improve everything. At least not now

Reply:
It's all good. I improved this page and I'll try to continue updating them as new issues arise involving affected code.